projects/elife.yaml

```diff
@@ -745,6 +747,57 @@ generic-cdn:
             default-ttl: 86400 # seconds
             headers:
                 - Origin
+        fastly:
+            subdomains:
+                - "{instance}-cdn"
+            backends:
+                # first is default
+                default:
+                    hostname: elife-cdn.s3.amazonaws.com
+                articles:
+                    hostname: "{instance}-elife-published.s3.amazonaws.com"
+                    condition: req.url ~ "^/articles/"
+            default-ttl: 86400 # seconds
+            surrogate-keys:
+                article-id:
+                    url: "^/articles/([0-9]+)/(.+)$"
+                    value: "articles/\\1"
```

Comment:
Should use `article/12345`, as we use that format elsewhere. Means it's a little different to the original URIs though (here, and our API).

Reply:
Where? Here I ported the original CDN paths but they seemed in line with https://elifesciences.org/articles/10627